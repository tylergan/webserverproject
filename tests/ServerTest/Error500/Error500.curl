curl 127.0.0.1:8070/cgibin/Error500.py