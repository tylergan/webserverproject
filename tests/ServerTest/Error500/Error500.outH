HTTP/1.1 500 Internal Server Error
Content-Type: text/html

