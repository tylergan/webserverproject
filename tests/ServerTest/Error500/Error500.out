<html>
<head>
	<title>500 Server Error</title>
</head>
<body bgcolor="white">
<center>
	<h1>500 Server Error</h1>
</center>
</body>
</html>
